utils/onnxmlir/src/onnxmlir/RunONNXModel.py

```diff
@@ -0,0 +1,959 @@
+#!/usr/bin/env python3
```

Comment:
Is this an identical copy? If so, at a minimum there should be a comment stating that they should stay in sync. Or maybe we could have a single version and we could "build" the pip structure by copying files in the right place. That would be my preference.

Reply:
Added a target in CMake to check the files.